doc/radosgw/notifications.rst

```diff
@@ -96,6 +96,7 @@ Topics
 .. note::
 
     In all topic actions, the parameters are URL encoded, and sent in the message body using ``application/x-www-form-urlencoded`` content type
```

Comment:
@zdover23 Orthogonal to this PR, I would put a _the_ after _using_ and omit the second comma. Also, the use of periods at the end of these sentences is inconsistent.

Reply:
@colemitchell: I'll make a PR tomorrow that addresses these usage and punctuation-related deficiencies.

Reply:
@colemitchell: https://github.com/ceph/ceph/pull/48493